common/source/docs/common-gps-for-yaw.rst

```diff
@@ -96,7 +95,21 @@ The Holybro Unicore UM982 GPS should have the following parameters set:
 
 .. note:: this unit can be used with only its "Master" antenna connected, if desired, but no yaw information should be used. In this case set :ref:`GPS_TYPE<GPS_TYPE>` = 24 (UnicoreMaster)
 
-For the Holybro UM982 GPS, and other Unicore UM982 GPSes, the "Master" and "Slave" antennas have more mounting flexibility, but must be mounted at least 30cm apart on the vehicle. The offset distances in the x/y/z directions must be entered for the following parameters:
+For the Holybro UM982 GPS, and other Unicore UM982 GPSes like the CUAV C-RTK2 HP, the "Master" and "Slave" antennas have more mounting flexibility, but must be mounted at least 30cm apart on the vehicle. The offset distances in the x/y/z directions must be entered detailed in the Master-Slave Antenna Offsets section below.
+
+CUAV DroneCAN C-RTK2 HP
+-----------------------
+
+The CUAV C-RTK2 HP unit can be connected to the autopilot either by serial or DroneCAN connections.
+
+See :ref:`CUAV C-RTK2 HP  Heading and RTK receiver<common-cuav-c-rtk2-hp>` for more information and setup details.
```

Comment:
The CUAV C-RTK2 HP  should have the following parameters set:
- :ref:`GPS_TYPE<GPS_TYPE>` = 9 (9:Dronecan；25：UART2)
- :ref:`EK3_SRC1_YAW <EK3_SRC1_YAW>` = 2 ("GPS") or 3 ("GPS with Compass Fallback") if a compass(es) is also in the system.

Reply:
I have added those to your c-rtk2 page: https://github.com/ArduPilot/ardupilot_wiki/pull/5199 since it has a dedicated page

Reply:
![image](https://github.com/ArduPilot/ardupilot_wiki/assets/31507501/1c1253c2-28f2-4985-9032-2cdcf838dae3)
I want c-rtk2 hp to be able to see all settings in one page instead of gps for yaw -> c-rtk2 hp (enable C-RTK2 HP) -> gps for yaw (antenna offset). This is bad experience.